oboe/oboe.h

```diff
@@ -1115,6 +1125,7 @@ int oboe_notifier_init(const char *socket_path);
 int oboe_notifier_stop(int blocking);
 int oboe_notifier_status();
 int oboe_notifier_test(int test_case, const char *test_str);
+int oboe_notifier_get(int what);
```

Comment:
how does the implementation look like for `oboe_notifier_get()`?

Reply:
this is also a leftover from a debug version of oboe and shouldn't be there.

i failed to put the production version of oboe into the oboe directory before i pushed. thanks for catching. i will fix tomorrow am.